autoblocks/_impl/global_state.py

```diff
@@ -44,3 +73,9 @@ def http_client() -> httpx.AsyncClient:
     if not _client:
         raise Exception("HTTP client not initialized")
     return _client
+
+
+def sync_http_client() -> httpx.Client:
+    if not _sync_client:
+        raise Exception("HTTP client not initialized")
+    return _sync_client
```

Comment:
This is used in tracer - 

On shutdown - using an async http client does not behave nicely, as asyncio does not allow you use coroutines as the program is exiting. 

I figure using sync client is okay here, as the http call happens in the background thread and won't block execution

Reply:
doesn't this defeat the purpose of using the async event loop?